models/Book.js

```diff
@@ -0,0 +1,18 @@
+const mongoose = require('mongoose');
+
+
+const bookSchema = new mongoose.Schema({
+    title:{
+        type: String,
+        required: true,
+    },
+    description:{
+        type: String,
+    },
+    timeCreated: {
```

Comment:
you can use the time stamp filed like
const bookSchema - new monoose.Schema ({
      ....
}, 
{
    toJSON : {
     // this will remove this fields from the response  json
        transform(doc, ret){
            delete ret.__v;
            delete ret.createdAt;
            delete ret.updatedAt;
        }
    },
    timestamps: true
}
)